tests/io/write_mesh_particles.cc

```diff
@@ -111,6 +112,23 @@ bool write_json(unsigned dim, bool resume, const std::string& analysis,
   return true;
 }
 
+// Write JSON Entity Set
+bool write_entity_set() {
+  // JSON Entity Sets
+  Json json_file = {
+      {"particle_sets",
+       {{{"id", 2},
+         {"set", {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}}}}}};
+
+  // Dump JSON as an input file to be read
+  std::ofstream file;
+  file.open("entity_sets_0.json");
```

Comment:
Is there already an `entity_sets.json`?

Reply:
yes, it consists of 2 node sets. I couldn't find where it was being generated from.